src/Routes.js

```diff
@@ -30,14 +31,19 @@ export default function Routes() {
             <Login />
           </Route>
           <ProtectedRoute
-            exact
-            path="/my-space"
-          >
-            <MySpace />
+            exact path="/my-portfolio">
```

Comment:
can't we just say /portfolio /profile ? this my my doesn't sound right

Reply:
Updated. I've removed `My`. Now it's only `Portfolio/Profile`
<img width="667" alt="Screen Shot 2020-07-13 at 1 03 16 am" src="https://user-images.githubusercontent.com/29667122/87249839-d8406180-c4a4-11ea-9da7-ef86eca8f288.png">